protocol-tests/src/main/java/module-info.java

```diff
@@ -1,5 +1,5 @@
-module fixtures.bodystring {
```

Comment:
Should we delete the module-info file in `generate` script?

In my case in fluent java8 not able to recognize it.
https://github.com/Azure/autorest.java/blob/v4/fluent-tests/prepare-tests.bat#L48-L49